cypress/commons/constants/generic/IdConstants.js

```diff
@@ -64,7 +64,7 @@ export const GENERIC_SELECTORS = {
       placeholder: '[data-cy=dashboard-placeholder]',
       accordion: '[data-cy=dashboards-accordion]',
       accordionSummary: '[data-cy=dashboards-accordion-summary]',
-      accordionLogsDownloadButton: '[data-cy=successful-run-logs-download-button]',
+      accordionLogsDownloadButton: '[data-cy=run-logs-download-button]',
```

Comment:
@nborde-CSM The previous value "_successful-run-logs-download-button_" for this selector is still used in _src/views/Scenario/components/ScenarioDashboardCard/ScenarioDashboardCard.js_, we should probably keep the previous one for scenario download logs and create a new selector for the runner logs download